### PR TITLE
Tests/test data

### PR DESCRIPTION
**Pull request by rebeckahendricks**
*Created a set of test data to use for ActiveRecord methods
*Created model tests for top_5_customers and items_ready_to_ship
*Created feature tests for merchant dashboard
(Test coverage is now at 100%)

### CODE REVIEW

File: spec/features/merchants/dashboard_spec.rb [hunk 1]
```diff
@@ -1,95 +1,186 @@
...
 
-        expect(Merchant.first.top_five_customers_by_merchant).to eq(expected_hash)
+        within ("#top_customers") do
+          customer1 = "Porter Whitehall"
+          customer2 = "Donna Petereit"
+          customer3 = "Carla Whipkey"
+          customer4 = "Becka Hendricks"
+          customer5 = "John Doe"
+
+          expect(customer1).to appear_before(customer2)
```
Comment (thayes87) [inline spec/features/merchants/dashboard_spec.rb:108]: Nice work with orderly

File: spec/features/merchants/dashboard_spec.rb [hunk 2]
```diff
@@ -1,95 +1,186 @@
...
-    end
 
-    xit 'Next to each customer name I see the number of successful transactions they have conducted with my merchant' do
-      visit merchant_dashboards_path(Merchant.first)
+      it 'Next to each customer name I see the number of successful transactions they have conducted with my merchant' do
+        visit merchant_dashboards_path(45)
 
-      expected_hash = {}
+        within "#customer_49" do
+          expect(page).to have_content("Porter Whitehall - 5 purchases")
```
Comment (thayes87) [inline spec/features/merchants/dashboard_spec.rb:119]: we should add a few .to_not have statements in these within blocks during refactoring

**Review by thayes87 — APPROVED** [review]
Thank you for taking charge with the test data, this will prove very helpful for the remainder of the project. My only feed back is to add .to_not have_content statements inside of the within statements. We easily and quickly address this during refactoring.

File: spec/features/merchants/dashboard_spec.rb (hunk 2 above)
Comment (lcole37) [reply spec/features/merchants/dashboard_spec.rb:119]: Test data looks great!!!!

**Review by lcole37 — APPROVED** [review]

**Review by Dominicod — APPROVED** [review]

**Pull request merged**